CrewChiefV4/UserInterface/MainWindow.cs

```diff
@@ -2343,6 +2361,32 @@ private void internetPanHandler(object sender, EventArgs e)
         {
             Process.Start("http://thecrewchief.org/misc.php?do=donate");
         }
+
+        // get the first decendant or null if we can't
+        public static String getElementDescendantValue(XElement parent, String elementName)
```

Comment:
good candidate methods for Utilties class

Reply:
Aye